lib/iris/experimental/representation.py

```diff
@@ -271,9 +276,17 @@ def _make_content(self):
                         title = body.pop(0)
                         colspan = 0
                     else:
-                        split_point = line.index(':')
-                        title = line[:split_point].strip()
-                        body = line[split_point + 2:].strip()
+                        try:
+                            split_point = line.index(':')
+                        except ValueError:
```

Comment:
I can see this works, but I've struggled to understand what this is really about, or what 'expand_last_cell' means + how it fits in.
Could you possibly add a comment to make clear in what kind of case it fails here (presumably missing a ':'), and what you are doing to resolve it ?

Reply:
The case that this fixes is when an attribute name contains a '\n' and no ':' beyond the '\n'. This causes a problem because we are reading data from the cube.summary. When the cube summary reads a '\n' it is creating a new line in the summary which has no ':'. Now that I think about it, it may actually be more sensible to escape any newlines in the cube.summary method.